lib/src/base-styles/_nx-card.scss

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-card-container {
+  display: flex;
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
+
+.nx-card {
+  background-color: #fff;
+  border-radius: $nx-border-radius;
+  box-sizing: border-box;
+  padding: $nx-spacing-l $nx-spacing-xxl;
+}
+
+.nx-card__call-out {
+  .nx-icon {
+    font-size: 48px;
+  }
+
+  .nx-icon--large {
+    font-size: 64px;
+  }
+}
+
+.nx-call__call-out-text {
+  @include bold();
+
+  font-size: 32px;
+}
+
+// Vertically oriented cards appear in a row container
+.nx-card-container--row {
+  @include container-horizontal;
+
+  .nx-card {
+    display: flex;
+    flex-direction: column;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    margin: 0 $nx-spacing-md 0 0;
+    text-align: center;
+    min-width: 230px;
+    max-width: 330px;
+  }
+
+  .nx-card-header {
+    @include container-vertical;
+    margin-bottom: $nx-spacing-md;
+  }
+
+  .nx-card__call-out {
+    margin-bottom: $nx-spacing-xs;
+  }
+
+  .nx-card__text {
+    margin-bottom: $nx-spacing-l;
+  }
+
+  .nx-card-footer {
```

Comment:
Is this element necessary or could these styles just be applied directly to the `.nx-text-link`.  That's the only thing that's ever supposed to be in the footer, right?

Reply:
> Is this element necessary or could these styles just be applied directly to the .nx-text-link. That's the only thing that's ever supposed to be in the footer, right?

Nevermind I see there's mockups where it has buttons instead.

Reply:
> Is this element necessary or could these styles just be applied directly to the `.nx-text-link`. That's the only thing that's ever supposed to be in the footer, right?

Well there are examples where there are buttons in the footer. I found with the grid layout that applying the grid to objects rather than containers caused all sorts of issues when it came to following the designs, and the also is obviously a problem if anybody ever adds multiple things in the footer.